### PR TITLE
temporary fix for increasing width of inputs

### PR DESCRIPTION
**Pull request by f-fromont**
slightly buggy, if you hold down a key it may not capture the length quite correctly, but it tends to work as long as you type in carefully.

Also sometimes breaks (doesn't capture length properly) if you type from earlier in the string

### CODE REVIEW

File: parsons-tool-frontend/src/components/blocks/Block.js [hunk 1]
```diff
@@ -74,7 +74,14 @@ const toFadedChildren = (text, fadedIndices, innerProps) => {
...
   const i = Math.floor(index / 2);
   return {
-    onChange: (e) => setInput(i, e.target.value),
+    onChange: (e) => {
+      setInput(i, e.target.value);
+      if (e.target.value.length > 10) {
+        e.target.style.width = `${e.target.value.length}ch`;
+      } else {
+        e.target.style.width = '10ch';
+      }
```
Comment (HiruNya) [inline parsons-tool-frontend/src/components/blocks/Block.js:83]: I'd consider making 10 characters the minimum that it could be but this also good.

**Review by HiruNya — APPROVED** [review]
LGTM

File: parsons-tool-frontend/src/components/blocks/Block.js (hunk 1 above)
Comment (f-fromont) [reply parsons-tool-frontend/src/components/blocks/Block.js:83]: wait could you clarify, with this condition 10ch is the minimum the width gets set to

Comment (HiruNya) [reply parsons-tool-frontend/src/components/blocks/Block.js:83]: Ohh yes, I misunderstood the code - I'd have just done

```
e.target.style.width = `${Math.min(10, e.target.value.length)}ch`;
```

but it's the same thing as what you have anyway.

Comment (f-fromont) [reply parsons-tool-frontend/src/components/blocks/Block.js:83]: rightio, I think I will keep the current version as its a bit more readable

**Pull request merged**